doc/install/requirements.md

```diff
@@ -0,0 +1,68 @@
+# Requirements
+
+## Operating Systems
+
+### Supported Unix distributions
+
+- Ubuntu (12.04 and 14.04)
+- Debian (Jessie and Wheezy)
+
+### Unsupported Unix distributions
+
+- CentOS
+- Red Hat Enterprise Linux
+- OS X
+- Arch Linux
+- Fedora
+- Gentoo
+- FreeBSD
+
+On the above unsupported distributions is still possible to install Huginn. Follow the [installation guide](./installation.md) and substitute the `apt` commands with the corresponding package manager commands of your distribution.
```

Comment:
Maybe "On the above unsupported distributions is still possible to install Huginn, and many people do."